packages-next/keystone/src/lib/core/field-assertions.ts

```diff
@@ -13,9 +13,9 @@ export function assertFieldsValid(list: ListForValidation) {
 function assertUniqueWhereInputsValid(list: ListForValidation) {
   for (const [fieldKey, { dbField, input }] of Object.entries(list.fields)) {
     if (input?.uniqueWhere) {
-      if (dbField.kind !== 'scalar' || (dbField.scalar !== 'String' && dbField.scalar !== 'Int')) {
+      if (dbField.kind !== 'scalar') {
```

Comment:
you may not need this

Reply:
This fails, when dbField is, for example, BigInt, that is scalar too, but throws the error.